lib/services/ios-debug-service.ts

```diff
@@ -207,7 +207,13 @@ export class IOSDebugService extends DebugServiceBase implements IPlatformDebugS
 			return this.getChromeDebugUrl(debugOptions, this._socketProxy.options.port);
 		} else {
 			this._socketProxy = await this.$socketProxyFactory.createTCPSocketProxy(this.getSocketFactory(device));
-			await this.openAppInspector(this._socketProxy.address(), debugData, debugOptions);
+			// delay opening the AppInspector until after the runtime has finished initializing the
+			// dynamically registered Chrome DevTools agent implementations in JavaScript using the tns-core-modules
+			// The workaround is necessary to prevent an unexpected timing crash introduced in the 3.3 modules
+			const inspectorStartDelay = 6000;
+			setTimeout(async () => {
+				await this.openAppInspector(this._socketProxy.address(), debugData, debugOptions);
```

Comment:
In case this operation fails, you'll receive unhandled rejection, can you please add `try { } catch()` around the call and log the error with $logger.

Reply:
Certainly, will also rebase to release.